docker/libstuff/bedrock.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
just curious if there is a reason to use this bash script instead of the systemd configs in the repo and systemd to run the process?

Reply:
systemd is never include into docker because of the single process recommandation of docker. If you take a little time to read [https://github.com/phusion/baseimage-docker](https://github.com/phusion/baseimage-docker) you will understand ;)

Reply:
and the second argument is that we need to change a bit the configuration to handle the env. var.:

From the Readme:

## Environment Variables

The following environment variables can be used to configure Bedrock:

- `NODE_NAME`: Name of the Bedrock node
- `PRIORITY`: Priority of the node in the cluster
- `PEER_LIST`: Comma-separated list of peer nodes
- `PLUGINS`: Plugins to enable
- `CACHE_SIZE`: Size of the cache
- `WORKER_THREADS`: Number of worker threads
- `QUERY_LOG`: Query log configuration
- `MAX_JOURNAL_SIZE`: Maximum journal size
- `SYNCHRONOUS`: Synchronous mode setting